docs/adguard-vpn-browser-extension/solving-problems/logs.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How to collect and send logs
+sidebar_position: 1
+---
+
+If you encounter a problem when using AdGuard VPN Browser Extension, you can inform us about it by sending application logs.
+
+## Collecting and sending standard logs
+
+By default, AdGuard VPN Browser Extension uses the Default logging level, that is, the basic collection of data about running processes of the app. To send these logs, follow the next steps:
+
+1. Open AdGuard VPN Browser Extension, if possible, repeat the actions that led to the error. Note the exact time when this error occurred.
+
+2. Open *Settings* by clicking the hamburger menu icon (☰) → *Support* → *Report a bug*.
+
+3. In the opened form, enter your email address for feedback and describe the error found, including the time when this error occurred. If you can't reproduce the problem, specify as accurately as possible when it last occurred.
```

Comment:
No need to enter an email. It's filled automatically (but it's possible to change email to another one if needed)